MelteCompiler.js

```diff
@@ -344,12 +344,10 @@ SvelteCompiler = class SvelteCompiler extends CachingCompiler {
             : { code: processedCode };
         },
         style: async ({ content, attributes }) => {
-          if (this.postcss) {
-            if (attributes.lang == 'postcss') {
-              return {
-                code: await this.postcss.process(content, { from: undefined })
-              };
-            }
+          if (this.postcss && (!attributes.lang || attributes.lang === 'postcss')) {
```

Comment:
For version 2 of melte-compiler, we should remove the checks for attribute.lang and always run postcss to match what svelte-preprocessor does (it runs the preprocessor for the lang if there is one, and then runs postcss).

To avoid introducing a backwards breaking change now, could we add an option for this behaviour?

Reply:
Sure, that's what I suggested in #5 

What would you like to config option to be named? I'm happy to update the PR.

Reply:
I'm not sure what a good name would be. Maybe something like `alwaysUsePostCSS`?